### ISSUE TITLE
Can't accept follow requests with Metro 2.0

### ISSUE THREAD

**Issue by MrAn0nym** [pixelfed/pixelfed#3530]
Using the new UI it's not possible to accept follow requests (at least on [pixelfed.social](pixelfed.social)). There is no visual feedback other than the request not being removed from the list. 

To busy rn to create another account for providing the exact error returned by the server, so if you have two accounts feel free to post more details below ^^

_(Error also mentioned in this comment https://github.com/pixelfed/pixelfed/issues/1594#issuecomment-1122419377)_

**Comment by MrAn0nym** [comment]
Did some further testing, seems like the client is not sending the id, which leads to a 500 error

Request body:
```json
{"action":"accept"}
```

Response:
```json
{"message":"The given data was invalid.","errors":{"id":["The id field is required."]}}
```

>

**Comment by dansup** [comment]
Fixed in #3532, any existing follow requests should be rejected and once your instance updates to the latest commits, you'll be able to properly accept remote follow requests
